gui/vm/forms.py

```diff
@@ -97,13 +123,34 @@ class DeviceForm(ModelForm):
         required=False,
         dirsonly=False,
     )
+    DISK_raw_boot = forms.BooleanField(
+        label=_('Disk boot'),
+        widget=forms.widgets.HiddenInput(),
+        required=False,
+        initial=False,
+    )
+    ROOT_password = forms.CharField(
+        label=_('Password'),
+        max_length=50,
+        widget=forms.widgets.HiddenInput(),
+        required=False,
+        help_text=_("Set the user rancher password."),
```

Comment:
Should it be 'Rancher' here?

Reply:
No, the username is 'rancher'.

Reply:
@wblock @araujobsd I am not a native speaker, but shouldn't it be `Set the "rancher" used password.`?

Reply:
Ah.  I'd say it would be clearer to say "Set the password for the "rancher" user."